crypto/src/stake/penalty.rs

```diff
@@ -0,0 +1,108 @@
+use std::str::FromStr;
+
+use penumbra_proto::{core::stake::v1alpha1 as pbs, Protobuf};
+use serde::{Deserialize, Serialize};
+
+use crate::{asset, Amount, Balance, Value, STAKING_TOKEN_ASSET_ID};
+
+/// Tracks slashing penalties applied to a validator in some epoch.
+///
+/// The penalty is represented as a fixed-point integer in bps^2 (denominator 10^8).
+#[derive(Debug, Copy, Clone, PartialEq, Eq, Serialize, Deserialize)]
+#[serde(try_from = "pbs::Penalty", into = "pbs::Penalty")]
+pub struct Penalty(pub u64);
+
+impl Default for Penalty {
+    fn default() -> Self {
+        Penalty(0)
+    }
+}
```

Comment:
This Default impl lets us unconditionally pull Penalties out of the state without having to actually record them in the common case (no slashing).